recipe/meta.yaml

```diff
@@ -11,30 +11,37 @@ source:
 
 build:
   number: 0
+  skip: True  # [py<36]
   entry_points:
     - theano-cache = bin.theano_cache:main
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
   host:
     - pip
     - python
+    - setuptools
+    - wheel
+    - numpy
   run:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
     - python
-    - setuptools
-    - six >=1.9.0
-    - numpy >=1.9.1
+    - {{ pin_compatible('numpy') }}
     - scipy >=0.14
-    - pygpu >=0.7.0,<0.8
-    - jax  # [py>=37 and not win]
+    #- six >=1.9.0
```

Comment:
Add here a comment that we disable it until package jax gets available for all architectures.  AFAI know was/is DH working on it, and made progress.  But well, for now it looks to me fine to provide package without this optimization

Reply:
I added the comment